interactions/api/models/guild.py

```diff
@@ -360,6 +360,8 @@ def __attrs_post_init__(self):  # sourcery skip: last-if-guard
                     self.member_count = guild.member_count
                 if not self.presences:
                     self.presences = guild.presences
+                if not self.emojis:
+                    self.emojis = guild.emojis
```

Comment:
Please also add this to adding it to the global cache as above done with other objects

Reply:
Emojis doesn't exists in global cache.

Reply:
Why would they not?

Reply:
> Why would they not? 

Discord has only one event with emoji. Its `guild_emojis_update`. On unstable it doesn't gets emojis and not stores them in cache.
To cache emojis we should get emojis from `GuildEmojis` object and do some stuff with cache. Same for stickers.